lightning/restore/restore.go

```diff
@@ -153,6 +154,10 @@ type RestoreController struct {
 }
 
 func NewRestoreController(ctx context.Context, dbMetas []*mydump.MDDatabaseMeta, cfg *config.Config) (*RestoreController, error) {
+	return NewRestoreControllerWithPauser(ctx, dbMetas, cfg, DeliverPauser)
```

Comment:
Shall we rename `DeliverPauser` as `DefaultPauser`?  cc @kennytm

Reply:
Either is fine.

Reply:
Do you know why the test remains "Pending" for hours?

Reply:
Jenkins config issue. Fixing it @suzaku